replay.go

```diff
@@ -113,17 +114,34 @@ func ParseReplay(file []byte) (r *Replay, err error) {
 		}
 	}
 
-	if b.Len() == 8 {
-		if r.ScoreID, err = rInt64(b); err != nil {
-			return nil, fmt.Errorf("reading ScoreID: %s", err)
+	if r.ScoreID, err = rInt64(b); err != nil {
```

Comment:
Older stable replays had a 32bit score id so there should be a fallback for it that was removed.

Reply:
I'll fix this when I'm back from my trip, cheers